v3-nginx/Dockerfile

```
@@ -84,34 +89,33 @@ COPY --from=build /usr/share/TLS/server.key /etc/nginx/conf/server.key
 COPY --from=build /usr/share/TLS/server.crt /etc/nginx/conf/server.crt
 COPY ./tls.conf /etc/nginx/conf.d/tls.conf.disabled
 
-RUN export version=$(echo `/usr/sbin/nginx -v 2>&1` | cut -d '/' -f 2) && \
-    wget --quiet http://nginx.org/download/nginx-$version.tar.gz && \
-    tar -xvzf nginx-$version.tar.gz && \
-    cd /nginx-$version && \
+RUN version="$(/usr/sbin/nginx -v 2>&1 | cut -d '/' -f 2)" && \
+    wget --quiet http://nginx.org/download/nginx-"$version".tar.gz && \
+    tar -xvzf nginx-"$version".tar.gz && \
+    cd /nginx-"$version" && \
     ./configure --with-compat --add-dynamic-module=../ModSecurity-nginx && \
     make modules && \
     cp objs/ngx_http_modsecurity_module.so /etc/nginx/modules
 
 RUN mkdir /etc/modsecurity.d && \
-    cd /etc/modsecurity.d && \
-    echo "include \"/etc/modsecurity.d/modsecurity.conf\"" > setup.conf && \
-    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/modsecurity.conf-recommended && \
-    mv modsecurity.conf-recommended modsecurity.conf && \
-    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/unicode.mapping
+    echo "include \"/etc/modsecurity.d/modsecurity.conf\"" > /etc/modsecurity.d/setup.conf && \
```

How about?
```suggestion
    echo 'include "/etc/modsecurity.d/modsecurity.conf"' > /etc/modsecurity.d/setup.conf && \
```
or even
```suggestion
    echo "include /etc/modsecurity.d/modsecurity.conf" > /etc/modsecurity.d/setup.conf && \
```

---

Excellent.